test/linarith.lean

```diff
@@ -123,9 +138,6 @@ example (x y : ℚ)
   (h : 6 + ((x + 4) * x + (6 + 3 * y) * y) = 3 ∧ (x + 4) * x ≥ 0 ∧ (6 + 3 * y) * y ≥ 0) : false :=
 by linarith
 
-example (x y : ℕ) (h : 6 + ((x + 4) * x + (6 + 3 * y) * y) = 3) : false :=
```

Comment:
Was `linarith` not supposed to solve this one before?

Reply:
The better zification causes this to fail, since it no longer recognizes that some products of nats are nonnegative. There's a tradeoff here -- it will solve some additional things and fail on some things that it solved before.